tests/assertions/native-functions.n

```diff
@@ -63,7 +63,7 @@ assert value range(0, 0, 1) = []
 assert value range(1, 0, 1) = []
 
 // `type` is not a "function" per se
```

Comment:
We can remove this now
```suggestion
```

Though do we need `getType`? What would the use case be?

Reply:
Ummm, nothing really

Reply:
Ok, I removed it and `getType`